lib/base/error-handler.js

```diff
@@ -21,6 +24,7 @@ const log = require('npmlog');
             "Cannnot install privileged app on developer mode" : "Please change the app id (app id should not start with 'com.lge', 'com.webos', 'com.palm')",
             "Cannnot remove privileged app on developer mode" : "You cannot remove the privileged app whose app id start with 'com.lge', 'com.webos', 'com.palm'",
             "unable to execute smack" : "Please remove the app and install again",
+            "empty session list, cannot query getAppInfo" : "Please select \"Profile icon\" on display",
```

Comment:
how about 
Select a profile on each display.

Reply:
I'll change it to "Please select the profile on the display"